guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+
+
+       $ PATH=$PATH:$HOME/.local/bin:$HOME/bin:${SQREAM_HOME}/bin/:${JAVA_HOME}/bin:$HADOOP_INSTALL/bin
+       $ export PATH
+
+3. Verify that the edits have been made:
+
+     .. code-block:: console
+     
+       $ source /home/sqream/.bash_profile
+       
+4. Check if you can access Hadoop from your machine:       
+       
+   .. code-block:: console
+     
+      $ hadoop fs -ls hdfs://<hadoop server name or ip>:8020/
+      
+   **NOTICE:** If you cannot access Hadoop from your machine because it uses Kerberos, see [Connecting a SQream Server to Cloudera Hadoop with Kerberos].(https://sqream.atlassian.net/l/c/31tQvsrB)
+   
+5. Verify that an HDFS environment exists for SQream services:
+
+   .. code-block:: console
+     
+      $ ls -l /etc/sqream/sqream_env.sh
+      
+6. If an HDFS environment does not exist for SQream services, create one (sqream_env.sh): **Comment - Is sqream_env.sh the file name?**
+
+   .. code-block:: console
+     
+      $ #!/bin/bash
+
+      $ SQREAM_HOME=/usr/local/sqream
+      $ export SQREAM_HOME
+
+      $ export JAVA_HOME=${SQREAM_HOME}/hdfs/jdk
+      $ export HADOOP_INSTALL=${SQREAM_HOME}/hdfs/hadoop
+      $ export CLASSPATH=`${HADOOP_INSTALL}/bin/hadoop classpath --glob`
+      $ export HADOOP_COMMON_LIB_NATIVE_DIR=${HADOOP_INSTALL}/lib/native
+      $ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${SQREAM_HOME}/lib:$HADOOP_COMMON_LIB_NATIVE_DIR
+
+
+      $ PATH=$PATH:$HOME/.local/bin:$HOME/bin:${SQREAM_HOME}/bin/:${JAVA_HOME}/bin:$HADOOP_INSTALL/bin
+      $ export PATH
+      
+ 7. Start the following SQream services manually and verify that they are functioning correctly:     
+      
+    .. code-block:: console
+     
+       $ sudo systemctl start metadataserver
+       $ sudo systemctl start serverpicker
+       $ sudo systemctl start sqream1
+       $ sudo systemctl start sqream2
+       $ sudo systemctl start sqream3
+       $ sudo systemctl start sqream4  
+      
+ 8. Verify that the following SQream processes are running and listening:
+ 
+    .. code-block:: console
+     
+       $ sudo systemctl status metadataserver
+       $ sudo systemctl status serverpicker
+       $ sudo systemctl status sqream1  #... etc
+       $ sudo netstat -nltp   #to see that sqream is listening on ports
+      
+   **NOTICE:** - Depending on the GPU package build optimization, it may take sqreamd 3 - 5 minutes to begin listening on its ports. To verify that sqreamd has begun listening, view the service logs located in the **/var/log/sqream** directory.   
+      
+Installing Monit
+====================================
 
+**To install Monit:**   
+   
+1. Install Monit as a superuser:
+ 
+    .. code-block:: console
+     
+       $ sudo yum install monit  
+       
+**Comment - This was the only step in the Confluence page. Does the user have to do anything else to install Monit? Is there a procedure in some other place that is already written?**
```

Comment:
Great point. Furthermore, the command written to install Monit applies to CentOs. We should get the equivalent command for other environments such as Ubuntu.

We should optionaly have a link to a site that details how to install it in an offline mode (no Internet connection)

Reply:
On Ubuntu the command is 
sudo apt-get install monit
or
sudo apt install monit

Installing offline:
https://mmonit.com/wiki/Monit/Installation

Pre-built binaries

Pre-built Monit binaries can be found here. Example installation:

 $ tar zxvf monit-x.y.z-linux-x64.tar.gz (x.y.z denotes version numbers)
 $ cd monit-x.y.z
 $ cp bin/monit /usr/local/bin/
 $ cp conf/monitrc /etc/